top-sites/sites.js

```diff
@@ -0,0 +1,23 @@
+browser.topSites.get()
+  .then((sites) => {
+    var div = document.getElementById('site-list');
+
+    if (!sites.length) {
+      div.innerText = 'No sites returned from the topSites API.';
+      return;
+    }
+
+    let ul = document.createElement('ul');
+    ul.className = 'list-group';
+    for (let k in sites) {
```

Comment:
There were some more features I added, then I figured the code was too complicated, removed it and forgot to fix it back up.